app/views/admin/school_groups/_issues.html.erb

```diff
@@ -11,16 +11,16 @@
     </tr>
   </thead>
   <tbody>
-    <% school_group.issues.issue.status_open.by_pinned.by_updated_at.each do |issue| %>
+    <% school_group.school_issues.issue.status_open.by_pinned.by_updated_at.each do |issue| %>
       <tr>
-        <td><%= fa_icon('exclamation-circle text-secondary') %></td>
-        <td><%= link_to issue.title, admin_school_issue_path(issue.school, issue) %> <%= fa_icon('thumbtack text-secondary') if issue.pinned? %></td>
-        <td><%= link_to issue.school.name, admin_school_issues_path(issue.school) %></td>
-        <td><%= render 'admin/schools/issues/fuel_type', issue: issue %></td>
-        <td><%= render 'admin/schools/issues/owned_by', issue: issue %></td>
+        <td><%= issue_type_icon(issue.issue_type) %></td>
+        <td><%= link_to issue.title, polymorphic_path([:admin, issue.issueable, issue]) %> <%= fa_icon('thumbtack text-secondary') if issue.pinned? %></td>
+        <td><%= link_to issue.issueable.name, polymorphic_path([:admin, issue.issueable, Issue]) %></td>
```

Comment:
just to double check, should this be `issue` and not `Issue` on this line?  (lines 17 above and 23 below have a lower case I)

Reply:
Yes it should be Issue - polymorphic_path uses it indicate to point to the index